packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -34,17 +45,54 @@
         </oc-list>
       </template>
     </oc-drop>
-    <template v-if="$_ocCollaborators_hasAdditionalPermissions">
-      <label v-if="!isAdvancedRoleSelected" class="oc-label oc-mt-s">
-        <translate>Additional permissions</translate>
-      </label>
-      <additional-permissions
-        :available-permissions="selectedRole.additionalPermissions"
-        :collaborators-permissions="collaboratorsPermissions"
-        :class="{ 'oc-mt-s': isAdvancedRoleSelected }"
-        @permissionChecked="checkAdditionalPermissions"
-      />
-    </template>
+    <oc-drop
+      ref="customPermissionsDrop"
+      data-testid="files-recipient-custom-permissions-drop"
+      class="files-recipient-custom-permissions-drop"
+      mode="manual"
+      target="#files-collaborators-role-button"
+    >
+      <template #special>
+        <translate tag="h4" class="files-recipient-custom-permissions-drop-title"
+          >User can
```

Comment:
`User can` is not an appropriate title I think, since sharing is not only about users. What about `Grant custom permissions` or `Custom permissions` (just repeating the title of the menu item from the role select)?